modules/querier/http.go

```diff
@@ -50,6 +57,21 @@ func (q *Querier) TraceByIDHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if r.Header.Get("Accepts") == "application/protobuf" {
+		level.Info(cortex_util.Logger).Log("msg", "received content type application/grpc")
```

Comment:
i don't think we want to log this everytime.  it will just become spam in the logs

Reply:
This will be removed before merging. Add for debugging.